telepresence/outbound/local.py

```diff
@@ -137,5 +137,8 @@ def launch_vpn(
 def _flush_dns_cache(runner: Runner):
     if runner.platform == "darwin":
         runner.show("Connected. Flushing DNS cache.")
-        pkill_cmd = ["sudo", "-n", "/usr/bin/pkill", "-HUP", "mDNSResponder"]
-        runner.check_call(pkill_cmd)
+        pkill_cmd = ["sudo", "-n", "/usr/bin/pkill", "-HUP", "mDNSResponder_Z"]
```

Comment:
Why the `_Z` at the end?

Reply:
Sorry about that, I've used it to test error condition. I've removed it.